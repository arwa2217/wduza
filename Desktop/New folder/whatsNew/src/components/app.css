:root {
  --primary: #03bd5d;
  --primary__light: #dcf0e8;
  --primary__lighter: #f3f3f3;
  --primary__dark: #18b263;
  --secondary: #58c171;
  --secondary__light: #e3f6f2;
  --default: #999999;
  --grey__light: #666666;
  --grey__dark: #19191a;
  --white: #fff;
  --black: #000;
  --orange__dark: #dd4409;
  --orange__lightsalmon: #fff3e8;
  --light: #f1f2fc;

  --top__bar: #2c2c2c;
  --left__sidebar: #2c2c2c;

  --link-color: #4d4de2;
  --link-font-size: 12px;
  --reply-window-color: #dcf0e6;
  --divider-color: #e5e9f0;
}

@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap");

html,
body {
  font-family: "Roboto", sans-serif !important;
  /*background: var(--white) !important;*/
  overflow: hidden !important;
  /*color: var(--default) !important;*/
  height: 100%;
  font-size: 15px;
}

button,
input,
optgroup,
select,
textarea {
  font-family: "Roboto", sans-serif !important;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  font-family: "Roboto", sans-serif !important;
  font-weight: 400;
}

b,
strong {
  font-weight: 700;
}

#root {
  height: 100%;
}

a:hover {
  color: var(--primary__dark);
}

button:focus {
  outline: none;
}

/* ==========================================
		COMPONENTS : BADGES
========================================== */

.badge {
  border-radius: 100%;
  margin-right: 11px;
  width: 8px;
  height: 8px;
  padding: 0;
}

.badge-primary {
  background-color: #18b263;
}
.badge-warning {
  background-color: #ec801c;
}
.badge-danger {
  background-color: #ca4c70;
}

/* ==========================================
		COMPONENTS : BADGES
========================================== */

/* ==========================================
		COMPONENTS : TEXT COLORS
========================================== */

.text-primary {
  color: var(--primary) !important;
}

/* ==========================================
		COMPONENTS : TEXT COLORS
========================================== */

/* ==========================================
		COMPONENTS : BUTTONS
========================================== */

.btn {
  font-weight: 100;
  padding: 7px 0.75rem;
  border-radius: 2px;
  font-size: 16px;
}

.btn:focus {
  box-shadow: none;
}

/* Button Primary */

.btn-primary {
  color: var(--white);
  background-color: var(--primary);
  border-color: var(--primary);
}
.btn-primary.disabled,
.btn-primary:disabled {
  color: var(--white);
  background-color: var(--primary);
  border-color: var(--primary);
}

.btn-primary:hover,
.btn-primary:focus,
.btn-primary.focus,
.btn-primary:not(:disabled):not(.disabled):active,
.btn-primary:not(:disabled):not(.disabled).active {
  color: var(--white);
  background-color: var(--primary__dark);
  border-color: var(--primary__dark);
}

.btn-primary:focus,
.btn-primary.focus,
.btn-primary:not(:disabled):not(.disabled).active:focus,
.btn-primary:not(:disabled):not(.disabled):active:focus,
.show > .btn-primary.dropdown-toggle:focus {
  /* box-shadow: 0 0 0 0.2rem var(--primary__light); */
  box-shadow: none;
}

/* Button Outline Primary */

.btn-outline-primary {
  color: var(--primary);
  border-color: var(--primary);
}
.btn-outline-primary:hover,
.btn-outline-primary:not(:disabled):not(.disabled).active,
.btn-outline-primary:not(:disabled):not(.disabled):active {
  color: var(--white);
  background-color: var(--primary);
  border-color: var(--primary);
}
.btn-outline-primary:focus,
.btn-outline-primary.focus,
.btn-outline-primary:not(:disabled):not(.disabled).active:focus,
.btn-outline-primary:not(:disabled):not(.disabled):active:focus,
.show > .btn-outline-primary.dropdown-toggle:focus {
  /* box-shadow: 0 0 0 0.2rem var(--primary__light); */
  box-shadow: none;
}

/* Btn Light */

.btn-light {
  color: var(--grey__light);
  background-color: var(--light);
  border-color: var(--light);
}

/* ==========================================
		COMPONENTS : BUTTONS ENDS
========================================== */

/* ==========================================
		COMPONENTS : FORMS STARTS
========================================== */

/* Input Group */

.input-group-text {
  background: #fff;
  border-right: 0;
  border-color: #adbacd;
  border-radius: 0;
}

.input-group > .form-control:focus + .input-group-append > .input-group-text {
  border-color: var(--primary__light);
  box-shadow: none;
}

.input-group > .form-control:disabled + .input-group-append > .input-group-text,
.input-group
  > .form-control[readonly]
  + .input-group-append
  > .input-group-text {
  background-color: #e9ecef;
  opacity: 1;
}

/* Form Control */

.form-control {
  /* border-radius: 0;
  border-color: #adbacd; */
  padding: 0.607142rem 0.75rem;
  height: auto;
  font-family: "Roboto", sans-serif !important;
}

.form-control:focus {
  border-color: var(--primary__light);
  box-shadow: none;
}

.form-control::placeholder {
  color: var(--default) !important;
}

.form-control:-ms-input-placeholder {
  /* Internet Explorer 10-11 */
  color: var(--default) !important;
}

.form-control::-ms-input-placeholder {
  /* Microsoft Edge */
  color: var(--default) !important;
}

/* ==========================================
		COMPONENTS : FORMS ENDS
========================================== */

/* ==========================================
		COMPONENTS : DROPDOWN STARTS
========================================== */

.dropdown-item.active,
.dropdown-item:active {
  background-color: #f0fbf5;
}

.dropdown-item:hover {
  background: rgba(3, 189, 93, 0.07);
}

/* ==========================================
		COMPONENTS : DROPDOWN ENDS
========================================== */

/* ==========================================
		COMPONENTS : CUSTOM SWITCH STARTS
========================================== */

.custom-switch {
  min-height: 18px !important;
}
.custom-switch .custom-control-label:before {
  content: "";
  width: 30px;
  height: 16px;
  left: -24px;
  background: #cecece;
}
.custom-switch .custom-control-input:checked ~ .custom-control-label:before {
  color: #fff;
  border-color: var(--primary);
  background-color: var(--primary);
}

.custom-switch
  .custom-control-input:disabled:checked
  ~ .custom-control-label:before {
  background-color: #dedede;
}

.custom-switch .custom-control-label::after {
  content: "";
  width: 14px;
  height: 14px;
  border-radius: 100%;
  top: 4px;
  left: -23px;
  background-color: #ffffff;
}
.custom-switch .custom-control-input:checked ~ .custom-control-label:after {
  background-color: #ffffff;
  transform: translateX(14px);
}
.custom-control-input:focus ~ .custom-control-label::before {
  box-shadow: none;
}
.custom-control-input:focus:not(:checked) ~ .custom-control-label::before {
  border-color: #cecece;
}

/* ==========================================
		COMPONENTS : CUSTOM SWITCH ENDS
========================================== */

/* ==========================================
		COMPONENTS : ALERTS STARTS
========================================== */

.custom-alert-modal {
  border-bottom: 20px solid rgba(0, 0, 0, 0.5);
}

.custom-alert-modal > .alert {
  margin-bottom: 0;
  padding: 14.5px;
}

/* ==========================================
		COMPONENTS : ALERTS ENDs
========================================== */

.vertical-center {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  padding-top: 48px;
  flex: 1;
  padding-bottom: 80px;
}

/* .onboarding-footer {
  position: absolute;
  bottom: 1rem;
  width: 100%;
} */

.message_bar {
  bottom: 1em;
  left: 0;
  right: 1em;
}
.app {
  /*background: var(--top__bar) !important;*/
}
.noScroll {
  overflow-x: hidden;
  overflow-y: hidden;
  height: 100vh;
}

.grid {
  margin: 0 !important;
  padding: 0 !important;
}

.ui.grid > * {
  padding: 0 !important;
}

.customColumn {
  padding: 7px 0 !important;
}

.ui.celled.grid > .row > .column {
  -webkit-box-shadow: 0 0 0 0 #d4d4d5 !important;
  box-shadow: 0 0 0 0 #d4d4d5 !important;
}

.ui.celled.grid > .row {
  -webkit-box-shadow: 0 0 0 0 #d4d4d5 !important;
  box-shadow: 0 0 0 0 #d4d4d5 !important;
}

/* Absolute Center Spinner */
.loading {
  position: fixed;
  z-index: 999;
  height: 2em;
  width: 2em;
  overflow: show;
  margin: auto;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}
.channel-type {
  padding-right: 25px;
}
.channel-type > div {
  justify-content: inherit;
}
/* Transparent Overlay */
.loading:before {
  content: "";
  display: block;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: radial-gradient(rgba(20, 20, 20, 0.6), rgba(0, 0, 0, 0.6));
  background: -webkit-radial-gradient(
    rgba(20, 20, 20, 0.6),
    rgba(0, 0, 0, 0.6)
  );
}
/*Accordion */
.ui.accordion .title:not(.ui) {
  font-family: "Roboto", sans-serif !important;
}
/* Animation */

@-webkit-keyframes spinner {
  0% {
    -webkit-transform: rotate(0deg);
    -moz-transform: rotate(0deg);
    -ms-transform: rotate(0deg);
    -o-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
    -moz-transform: rotate(360deg);
    -ms-transform: rotate(360deg);
    -o-transform: rotate(360deg);
    transform: rotate(360deg);
  }
}
@-moz-keyframes spinner {
  0% {
    -webkit-transform: rotate(0deg);
    -moz-transform: rotate(0deg);
    -ms-transform: rotate(0deg);
    -o-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
    -moz-transform: rotate(360deg);
    -ms-transform: rotate(360deg);
    -o-transform: rotate(360deg);
    transform: rotate(360deg);
  }
}
@-o-keyframes spinner {
  0% {
    -webkit-transform: rotate(0deg);
    -moz-transform: rotate(0deg);
    -ms-transform: rotate(0deg);
    -o-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
    -moz-transform: rotate(360deg);
    -ms-transform: rotate(360deg);
    -o-transform: rotate(360deg);
    transform: rotate(360deg);
  }
}
@keyframes spinner {
  0% {
    -webkit-transform: rotate(0deg);
    -moz-transform: rotate(0deg);
    -ms-transform: rotate(0deg);
    -o-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
    -moz-transform: rotate(360deg);
    -ms-transform: rotate(360deg);
    -o-transform: rotate(360deg);
    transform: rotate(360deg);
  }
}

/* :not(:required) hides these rules from IE9 and below */
.loading:not(:required) {
  /* hide "loading..." text */
  font: 0/0 a;
  color: transparent;
  text-shadow: none;
  background-color: transparent;
  border: 0;
}

.loading:not(:required):after {
  content: "";
  display: block;
  font-size: 10px;
  width: 1em;
  height: 1em;
  margin-top: -0.5em;
  -webkit-animation: spinner 150ms infinite linear;
  -moz-animation: spinner 150ms infinite linear;
  -ms-animation: spinner 150ms infinite linear;
  -o-animation: spinner 150ms infinite linear;
  animation: spinner 150ms infinite linear;
  border-radius: 0.5em;
  -webkit-box-shadow: rgba(255, 255, 255, 0.75) 1.5em 0 0 0,
    rgba(255, 255, 255, 0.75) 1.1em 1.1em 0 0,
    rgba(255, 255, 255, 0.75) 0 1.5em 0 0,
    rgba(255, 255, 255, 0.75) -1.1em 1.1em 0 0,
    rgba(255, 255, 255, 0.75) -1.5em 0 0 0,
    rgba(255, 255, 255, 0.75) -1.1em -1.1em 0 0,
    rgba(255, 255, 255, 0.75) 0 -1.5em 0 0,
    rgba(255, 255, 255, 0.75) 1.1em -1.1em 0 0;
  box-shadow: rgba(255, 255, 255, 0.75) 1.5em 0 0 0,
    rgba(255, 255, 255, 0.75) 1.1em 1.1em 0 0,
    rgba(255, 255, 255, 0.75) 0 1.5em 0 0,
    rgba(255, 255, 255, 0.75) -1.1em 1.1em 0 0,
    rgba(255, 255, 255, 0.75) -1.5em 0 0 0,
    rgba(255, 255, 255, 0.75) -1.1em -1.1em 0 0,
    rgba(255, 255, 255, 0.75) 0 -1.5em 0 0,
    rgba(255, 255, 255, 0.75) 1.1em -1.1em 0 0;
}

.scalePage {
  -webkit-transform: scale(0.77);
  -moz-transform: scale(0.77);
  -ms-transform: scale(0.77);
  transform: scale(0.77);
  color: #333;
}

.headerStyle {
  font-weight: 400 !important ;
  font-size: 48px;
}

.subHeaderStyle {
  font-size: 28px;
}

.subModalStyle {
  font-size: 22px;
  margin-bottom: 0.25rem;
}

.subHeaderStyleSignin {
  font-size: 22px;
}

.labelStyle {
  font-size: 17px;
}

.headerStyleMobile {
  margin-top: -30px;
  font-weight: 400 !important ;
  font-size: 40px;
}

.headerStyleMobileRegister {
  margin-top: -100px;
  font-weight: 400 !important ;
  font-size: 40px;
}

.subHeaderStyleMobile {
  font-size: 22px;
}

.labelStyleMobile {
  font-size: 12px;
}

::placeholder {
  color: #333 !important;
  opacity: 0.25 !important;
}

.customMargin {
  margin-top: -15px;
}

.text-upper-case {
  text-transform: uppercase;
}

.main-post-area {
  /*background: var(--top__bar);*/
}

:root {
  --link-color: #308f65;
  --link-font-size: 12px;
  --reply-window-color: #dcf0e6;
  --divider-color: #e5e9f0;
  --light-gray: #c8c8c8;
  --default-font-size: 15px;
  --light-gray2: #666666;
  --quill-link-font-size: 15px;
}

/* .ReplyView__replies .post.post-view.other--root .post__header .col__name {
  display: inline-block !important;
  overflow: visible;
} */

.custom-checkbox-green {
  margin-bottom: 12px;
}
.custom-control-input-green:focus ~ .custom-control-label::before {
  border-color: #28a745 !important;
  box-shadow: none !important;
}

.custom-control-input-green:checked ~ .custom-control-label::before {
  background-color: #fff;
}

.custom-control-input-green:checked ~ .custom-control-label::after {
  background-image: url(../assets/icons/checkbox-selected.svg) !important;
  background-repeat: no-repeat;
}

.custom-control-input-green:focus:not(:checked)
  ~ .custom-control-label::before {
  border-color: #5bd778 !important;
}

.custom-control-input-green:not(:disabled):active
  ~ .custom-control-label::before {
  background-color: #d6f5dd !important;
  border-color: #d6f5dd !important;
}

.custom-control-input {
  background: #ffffff;
  border: 1px solid #dedede;
  box-sizing: border-box;
  border-radius: 2px;
}

.custom-control-label {
  height: 14px;
  font-family: "Roboto", sans-serif;
  font-style: normal;
  font-weight: normal;
  font-size: 14px;
  line-height: 100%;
  color: #19191a;
  padding-top: 7px;
  padding-left: 10px;
  cursor: pointer;
}

.custom-checkbox
  .custom-control-input:indeterminate
  ~ .custom-control-label::before {
  background: #ffffff;
  border-color: #dedede;
}

.custom-checkbox-green > .custom-control-label::before {
  background: #ffffff;
  border: 1px solid #dedede !important;
  box-sizing: border-box;
  border-radius: 2px !important;
  width: 20px;
  height: 20px;
}

.custom-checkbox-green > .custom-control-label::after {
  width: 20px;
  height: 20px;
}

.custom-checkbox
  .custom-control-input:disabled:checked
  ~ .custom-control-label::before {
  background-color: #e0e0e0;
}

/* ==========================================
		COMPONENTS : CUSTOM RADIO
========================================== */

.custom-radio {
  display: flex;
}
.custom-radio > input[type="radio"] {
  display: none;
}
.custom-radio > label {
  display: inline-flex;
  color: #19191a;
  font-weight: 400;
  margin-bottom: 0;
  line-height: 20px;
  cursor: pointer;
}
.custom-radio > input[type="radio"] + label:before {
  content: "";
  display: inline-block;
  width: 20px;
  height: 20px;
  padding: 5px;
  margin-right: 10px;
  background-clip: content-box;
  border: 1px solid rgba(0, 0, 0, 0.5);
  background-color: #ffffff;
  border-radius: 50%;
}

.custom-radio input[type="radio"]:checked + label:before {
  background-color: #18b263;
}

/* ==========================================
		COMPONENTS : REACT AUTOSUGGEST
========================================== */

.react-autosuggest__container {
  position: relative;
  width: 100%;
}

.react-autosuggest__input {
  width: 240px;
  height: 30px;
  padding: 10px 20px;
  font-family: Helvetica, sans-serif;
  font-weight: 300;
  font-size: 16px;
  border: 1px solid #aaa;
  border-radius: 4px;
}

.react-autosuggest__input:focus {
  outline: none;
}

.react-autosuggest__container--open .react-autosuggest__input {
  border-bottom-left-radius: 0;
  border-bottom-right-radius: 0;
}

.react-autosuggest__suggestions-container {
  display: none;
}

.react-autosuggest__container--open .react-autosuggest__suggestions-container {
  display: block;
  position: absolute;
  top: 36px;
  width: 100%;
  /* border: 1px solid #aaa; */
  background-color: #fff;
  font-size: 14px;
  background: #ffffff;
  box-shadow: 0px 0px 4px rgba(76, 99, 128, 0.3);
  border-radius: 2px;
  z-index: 101;
  overflow-y: auto;
  max-height: 300px;
  word-break: break-all;
  overflow-x: hidden;
}

.react-autosuggest__suggestions-list {
  margin: 0;
  padding: 0;
  list-style-type: none;
  line-height: 1.6;
}

.react-autosuggest__suggestion {
  cursor: pointer;
  padding: 10px 12px;
  text-align: left;
}

.react-autosuggest__suggestion:not(:first-child) {
  border-top: 1px solid #ddd;
}

.react-autosuggest__suggestion--focused {
  background-color: #0c7eaf;
  color: #fff;
}

.suggestion-content {
  display: flex;
  align-items: center;
  background-repeat: no-repeat;
}

.highlight {
  color: #308f65;
}

.react-autosuggest__suggestion--focused .highlight {
  color: #120000;
}

.pointer-on-hover {
  cursor: pointer;
}
/* REACT AUTOSUGGEST END */

.notification-title-color {
  color: #19191a;
}

/* TOOLTIP STARTS */
.tooltip {
  font-family: "Roboto", sans-serif !important;
}
.tooltip > .tooltip-inner {
  padding: 3px 5px;
  line-height: 1.2;
}
/* TOOLTIP ENDS */

.global-search-result-view .message-post-wrap {
  padding: 0px;
  position: relative;
  bottom: unset;
}

/* page container styles */
.page-container {
  height: 100vh !important;
  overflow-y: auto;
}

/* app navigation*/
.menu-parent-icon .title {
  width: 100%;
}
.menu-parent-icon.title:not(.ui) {
  padding: 0;
}
.app-menu-link {
  display: flex;
  width: 100%;
  /*color: rgba(181, 181, 181, 1);*/
  white-space: nowrap;
  text-overflow: ellipsis;
  /*padding: 8px 0 8px 20px;*/
  cursor: pointer;
  /*border-radius: 8px;*/
  line-height: 1;
  overflow: hidden;
  padding-left: 16px;
  padding-top: 4px;
  padding-bottom: 4px;
  align-items: center;
}
.app-menu-link .menu-text {
  margin-left: 5px;
  padding-top: 2px;
}
.app-menu-link:hover {
  /*background: var(--black);
  color: var(--white);*/
}
.app-active-menu,
.app-active-menu:hover {
  /*background: var(--primary) !important;
  color: var(--white) !important;*/
}

/* home header */
.home-header {
  display: flex;
  margin-right: 10px;
}

.home-header .project-topbar {
  min-width: 300px;
  width: auto;
  height: 70px;
  background-color: #2c2c2c;
  align-items: center;
}

.home-header .notification-top-area {
  min-width: calc(100% - 300px);
  width: auto;
  background-color: var(--top__bar);
  padding: 16px 0;
  display: flex;
  flex-direction: row-reverse;
}

.home-header .notification-top-area .topBar__icon {
  margin-right: 5px;
  width: 38px;
  height: 38px;
  line-height: 40px;
  text-align: center;
  border-radius: 10px;
  font-size: 14px;
}
.home-header .notification-top-area .topBar__icon:not(.disabled) {
  cursor: pointer;
}
.home-header .notification-top-area .topBar__icon.topBar__icon__summary {
  background-color: transparent;
  line-height: 38px;
}
.home-header .notification-top-area .topBar__icon:not(.disabled):hover,
.home-header .notification-top-area .topBar__icon:not(.disabled):focus,
.home-header .notification-top-area .topBar__icon:not(.disabled).active {
  background-color: #000;
}
.home-header
  .notification-top-area
  .topBar__icon.topBar__icon__summary#opened:hover {
  background-color: #000;
}

/* Top Bar Notifications */
.home-header .notification-top-area .notification-icon {
  display: block;
  width: 38px;
  height: 38px;
  line-height: 38px;
  position: relative;
}
.home-header .notification-top-area .notification-icon > .badge {
  width: 8px;
  height: 8px;
  border-radius: 100%;
  position: absolute;
  top: 2px;
  right: 2px;
}
.home-header .notification-top-area .dropdown-menu {
  width: 500px;
  transform: translate3d(-460px, 50px, 10px) !important;
  padding: 0;
  border: 0;
  box-shadow: 0px 6px 5px rgba(0, 0, 0, 0.24), 0px 9px 18px rgba(0, 0, 0, 0.18);
}

.table {
  color: #19191a;
}

/* =============================================
  UTILS : Utility Classes
================================================*/

.pd-20 {
  padding: 20px !important;
}

/* =============================================
  Components : Download Status
================================================ */

.download-status {
  padding: 3px 10px;
  border-radius: 20px;
  font-weight: normal;
  font-size: 12px;
  line-height: 100%;
  display: inline-block;
  color: #ffffff;
}
.download-status.in {
  background: #c8c8c8;
}
.download-status.out {
  background: #ca4c70;
}

/* ==========================================
		COMPONENTS : MODALS
========================================== */

.custom-modal .modal-dialog {
  max-width: 660px;
}
.custom-modal .modal-content {
  border-radius: 8px;
}

.dropdown-icon {
  position: absolute;
  top: 3px;
  right: 0px;
}
.custom-modal .modal-header {
  display: flex;
  align-items: center;
  color: #252534;
  font-size: 20px;
  font-weight: 400;
  padding: 40px 40px 30px;
  border: none;
  border-radius: 0;
}

.custom-modal .modal-body {
  padding: 0 40px;
  max-height: 350px;
  overflow: auto;
}
.custom-modal .modal-footer {
  padding: 40px;
  border: 0;
}
.custom-modal .modal-footer .btn {
  min-width: 130px;
}
.custom-modal .form-label {
  color: #19191a;
  font-size: 15px;
  margin-bottom: 0;
}
.custom-modal .form-control {
  /* border-color: #ADBACD; */
  padding: 3.75px 10px !important;
  margin: 0 20px;
}
/* .custom-modal .input-group > .form-control{
    margin: 0 20px;
} */
/* .custom-modal .input-group{
    margin: 0 20px;
} */
.custom-modal .input-group > .input-group-text {
  border-left: 0;
  border-right: 1px solid #adbacd;
  border-color: #adbacd;
  padding: 0 10px;
}

/* available UID form */

.available-uid .input-btn {
  width: 80%;
  padding-right: 20px;
}
.available-uid .input-btn ::placeholder {
  color: #999999 !important;
  font-size: 15px !important;
}
.uid-close {
  opacity: 1 !important;
}
.available-uid .uid-label {
  width: 15%;
}

/* ==========================================
		COMPONENTS : TABS
========================================== */

.custom-tabs .nav-tabs .nav-item {
  color: #999999;
  padding: 10px 0px 5px 0px;
  margin-right: 20px;
}
.custom-tabs .nav-tabs .nav-link {
  border-bottom: 1px solid #dedede;
}
.custom-tabs .nav-tabs .nav-link:focus,
.custom-tabs .nav-tabs .nav-link:hover {
  border-color: transparent;
}
.custom-tabs .nav-tabs .nav-item.show .nav-link,
.custom-tabs .nav-tabs .nav-link.active {
  color: #19191a !important;
  background: #f8f8f8 !important;
  border-color: transparent transparent #03bd5d !important;
  border-bottom: 2px solid #03bd5d !important;
}

/* ==========================================
    COMPONENTS : SMALL BUTTON
========================================== */
.btn-small {
  font-size: 12px;
  padding: 0px !important;
  height: 30px;
}
.modal-add_user .modal-dialog {
  max-width: 288px;
  margin: auto;
}

.forward-e-sign-top
  .react-autosuggest__suggestions-container.react-autosuggest__suggestions-container--open {
  top: 24px;
}
