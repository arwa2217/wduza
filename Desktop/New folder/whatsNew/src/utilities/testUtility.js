import React from "react";
import { Provider } from "react-redux";
import { mount } from "enzyme";
import configureMockStore from "redux-mock-store";
import thunk from "redux-thunk";
import initialAppState from "../store/initialAppState";

const mockStore = configureMockStore([thunk]);

const store = mockStore(initialAppState);

export default {
  mount: (Component, props = {}, localState) => {
    let localStore = store;
    if (localState) {
      localStore = mockStore(localState);
    }
    const wrapper = mount(
      <Provider store={localStore}>
        <Component {...props} />
      </Provider>
    );
    return wrapper.find(Component).first();
  },
};
