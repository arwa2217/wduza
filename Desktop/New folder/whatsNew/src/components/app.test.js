import React from "react";
import { Router } from "react-router-dom";
import { App } from "./app";
import testUtility from "../utilities/testUtility";

test("App should have child router", () => {
  const wrapper = testUtility.mount(App);
  expect(wrapper.find(Router)).toHaveLength(1);
});

test("should match the snapshot", () => {
  const wrapper = testUtility.mount(App);
  expect(wrapper).toMatchSnapshot();
});
