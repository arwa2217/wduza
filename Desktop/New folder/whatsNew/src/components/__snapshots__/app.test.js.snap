// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<App>
  <Router
    history={
      Object {
        "action": "REPLACE",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "key": "irx58t",
          "pathname": "/home",
          "search": "",
          "state": Object {
            "from": Object {
              "hash": "",
              "pathname": "/",
              "search": "",
              "state": undefined,
            },
          },
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Switch>
      <Route
        component={[Function]}
        computedMatch={
          Object {
            "isExact": true,
            "params": Object {},
            "path": "/home",
            "url": "/home",
          }
        }
        location={
          Object {
            "hash": "",
            "key": "irx58t",
            "pathname": "/home",
            "search": "",
            "state": Object {
              "from": Object {
                "hash": "",
                "pathname": "/",
                "search": "",
                "state": undefined,
              },
            },
          }
        }
        path="/home"
      >
        <withI18nextTranslation(HomePage)
          history={
            Object {
              "action": "REPLACE",
              "block": [Function],
              "createHref": [Function],
              "go": [Function],
              "goBack": [Function],
              "goForward": [Function],
              "length": 1,
              "listen": [Function],
              "location": Object {
                "hash": "",
                "key": "irx58t",
                "pathname": "/home",
                "search": "",
                "state": Object {
                  "from": Object {
                    "hash": "",
                    "pathname": "/",
                    "search": "",
                    "state": undefined,
                  },
                },
              },
              "push": [Function],
              "replace": [Function],
            }
          }
          location={
            Object {
              "hash": "",
              "key": "irx58t",
              "pathname": "/home",
              "search": "",
              "state": Object {
                "from": Object {
                  "hash": "",
                  "pathname": "/",
                  "search": "",
                  "state": undefined,
                },
              },
            }
          }
          match={
            Object {
              "isExact": true,
              "params": Object {},
              "path": "/home",
              "url": "/home",
            }
          }
        >
          <HomePage
            history={
              Object {
                "action": "REPLACE",
                "block": [Function],
                "createHref": [Function],
                "go": [Function],
                "goBack": [Function],
                "goForward": [Function],
                "length": 1,
                "listen": [Function],
                "location": Object {
                  "hash": "",
                  "key": "irx58t",
                  "pathname": "/home",
                  "search": "",
                  "state": Object {
                    "from": Object {
                      "hash": "",
                      "pathname": "/",
                      "search": "",
                      "state": undefined,
                    },
                  },
                },
                "push": [Function],
                "replace": [Function],
              }
            }
            i18n={Object {}}
            location={
              Object {
                "hash": "",
                "key": "irx58t",
                "pathname": "/home",
                "search": "",
                "state": Object {
                  "from": Object {
                    "hash": "",
                    "pathname": "/",
                    "search": "",
                    "state": undefined,
                  },
                },
              }
            }
            match={
              Object {
                "isExact": true,
                "params": Object {},
                "path": "/home",
                "url": "/home",
              }
            }
            t={[Function]}
            tReady={false}
          >
            <div
              className="vertical-center container horizontal-center"
            >
              <div
                className="row justify-content-center mt-5"
              >
                <Container
                  className="text-center"
                  fluid={false}
                >
                  <div
                    className="text-center container"
                  >
                    <h1
                      className="d-none d-sm-block display-1 "
                    >
                      monoly.label
                    </h1>
                    <h1
                      className="d-block d-sm-none display-1 "
                    >
                      monoly.label
                    </h1>
                  </div>
                </Container>
              </div>
              <div
                className="row"
              >
                <div
                  className="col-lg-12 text-center mt-3"
                >
                  <p
                    className="d-none d-sm-block display-5 subHeaderStyle"
                  >
                    home.message.label
                  </p>
                  <p
                    className="d-block d-sm-none display-5 subHeaderStyleMobile"
                  >
                    home.message.label
                  </p>
                </div>
              </div>
              <div
                className="row  text-center mt-5"
              >
                <div
                  className="col-lg-3 offset-lg-3 text-center mt-5"
                >
                  <Link
                    className="btn btn-lg btn-outline-primary btn-block badge-pill customButtonHeight"
                    to="/signin"
                  >
                    <LinkAnchor
                      className="btn btn-lg btn-outline-primary btn-block badge-pill customButtonHeight"
                      href="/signin"
                      navigate={[Function]}
                    >
                      <a
                        className="btn btn-lg btn-outline-primary btn-block badge-pill customButtonHeight"
                        href="/signin"
                        onClick={[Function]}
                      >
                        <p
                          className="centerText"
                        >
                          signin.label
                        </p>
                      </a>
                    </LinkAnchor>
                  </Link>
                </div>
                <div
                  className="col-lg-3 offset-right-lg-2 text-center mt-5"
                >
                  <a
                    className="btn btn-primary btn-lg btn-block badge-pill customButtonHeight"
                  >
                    <p
                      className="centerText"
                    >
                      signup:label
                    </p>
                  </a>
                </div>
              </div>
            </div>
          </HomePage>
        </withI18nextTranslation(HomePage)>
      </Route>
    </Switch>
  </Router>
</App>
`;
